tracer/test/test-applications/integrations/Samples.Remoting/Program.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Linq;
+using System.Runtime.Remoting;
+using System.Runtime.Remoting.Channels;
+using System.Runtime.Remoting.Channels.Http;
+using System.Runtime.Remoting.Channels.Ipc;
+using System.Runtime.Remoting.Channels.Tcp;
+
+namespace Samples.Remoting
+{
+    public static class Program
+    {
+        public static void Main(string[] args)
+        {
+            string port = args.FirstOrDefault(arg => arg.StartsWith("Port="))?.Split('=')[1] ?? "9000";
+            Console.WriteLine($"Port {port}");
+
+            string protocol = args.FirstOrDefault(arg => arg.StartsWith("Protocol="))?.Split('=')[1]?.ToLower() ?? "http";
+            Console.WriteLine($"Protocol {protocol}");
+
+            string url = protocol switch
+            {
+                "http" => SetupHttpRemoting(port),
+                "tcp" => SetupTcpRemoting(port),
+                "ipc" => SetupIpcRemoting(port),
+                _ => throw new ArgumentException($"Protocol '{protocol}' not recognized", "Protocol"),
+            };
+
+            Console.WriteLine("Starting client calls");
```

Comment:
I wonder, is it worth adding a manual "wrapper" span on the client side to confirm we propagate the span context correctly, and a span on the server side for the same reason i.e.
manual -> client -> server -> manual ?

Reply:
Done in e7967d3a14ed760963c0a8b39fb2406b1659deab